packages/block-library/src/search/index.php

```diff
@@ -80,7 +80,7 @@ function render_block_core_search( $attributes ) {
 		}
 
 		$button_markup = sprintf(
-			'<button type="submit"class="wp-block-search__button ' . $button_classes . '">%s</button>',
+			'<button type="submit" class="wp-block-search__button ' . $button_classes . '">%s</button>',
```

Comment:
It looks like this change was accidentally included. I saw that it's being fixed in another PR so it's probably fine 😄